src/Components/GenreToggle.js

```diff
@@ -0,0 +1,25 @@
+import React, {useState} from 'react';
+import '../Styles/GenreToggle.css';
+
+function GenreToggle(props) {
+    const types = ["All", "Comedy", "Documentaries", "Fiction"];
+    const [active, setActive] = useState(types[0]);
+    return (
+        <div>
+            <h2>{props.title}</h2>
+            <div className="toggle-group">
+                {types.map(type => (
+                    <button
+                        key={type}
+                        className= {`toggle-tab ${active === type ? 'active' : null}`}
```

Comment:
Not a bug.
When condition is falsy, your class will render as `toggle-tab null`.
To avoid that - just replace `null` with `empty string`.
In real world app you would use some `npm` dependency to handle this for you. E.g. `classnames`